### DISCUSSION TITLE
Download manual?

### DISCUSSION THREAD

**Discussion by chrisjj** [sumatrapdfreader/sumatrapdf#4643] [Q&A]
How can I download the manual?

When I click Help > Manual, I am disappointed to be take to a web page ... if and only if I am online.

**Comment by GitHubRulesOK** [comment]
The pre-release "should" download the current manual for F1 usage (it has done previously) 
Unsure about current use as non functional in my odd current mix (but not checked why as in my case that will need reboot computer)

**Reply by chrisjj** [reply]
Thanks. I assume the answer is "you cannot".

**Comment by GitHubRulesOK** [comment]
No "it works for me"
I said I need to reboot because I run multiple conflicting installs so on fresh down load latest and press F1 it works with local manual

![image](https://github.com/user-attachments/assets/5260ff70-972d-4c05-bf3b-c892ded70de8)
